tests/unistd/unlink.c

```diff
@@ -150,7 +136,7 @@ void test() {
   err = access("file1", F_OK);
   assert(!err);
 #endif
-#ifndef NO_SYMLINK
+#if !defined(NO_SYMLINK) && !defined(WASMFS)
```

Comment:
Add a TODO with a link to a bug maybe?

Reply:
I already have the PR to remove this all ready to go... :smile: